tests/asset-css.test.js

```diff
@@ -477,6 +477,7 @@ tap.test('Css() - set "href" - should throw', (t) => {
 
 tap.test('Css() - validate object against schema - should validate', (t) => {
     const obj = new AssetCss({ value: '/foo' });
+    // @ts-expect-error TODO: should the schema accept boolean for crossorigin?
```

Comment:
Not sure if we should update the schema here or not

Reply:
hmm good question, its boolean at the object level true, but this is the same as crossorigin="". Feels like a the schema level, it should follow the spec https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin
 "", "use-credentials" or "anonymous"